README.md

```diff
@@ -18,6 +18,16 @@ Install MQTT Plugin:
 
 If you want to learn more about Spin's plugin model, read [here](https://www.fermyon.com/blog/managing-spin-templates-and-plugins).
 
+### Build and install from source
+
+You will need Rust and the pluginify plugin:
+
+```
+spin plugins install pluginify
```

Comment:
I think that's a great idea. Cc/ @itowlson any objections?

Reply:
It shouldn't belong to SpinKube.

We could move it under the Fermyon org but I'd like to understand the need.  Is it the bus factor thing?  Or is it causing a problem that the downloads come from an individual GH account rather than a corporate GH account?  (Even though they go through the Fermyon registry.)  I don't want to get into a mindset of "plugins should be hosted in Fermyon repos to be official" kind of thing...

Reply:
@itowlson mostly bus factor thing and I guess better visibility for external people to make use of it. I ended up having this solved in the [make file](https://github.com/spinkube/spin-mqtt-trigger-sdk/blob/main/Makefile) for example, ideally I'd should have used pluginify.